controller/translationcontroller.php

```diff
@@ -314,17 +314,17 @@ public function getLanguageStrings() {
 
 			// templates/views/show_vault.html
 			'deleted.since' => $this->trans->t('Showing deleted since'),
-			'alltime' => $this->trans->t('All time'),
+			'alltime' => $this->trans->t('Beginning'),
```

Comment:
I think translations of variants for "always" would have the issue "All time" has in English. "Beginning" makes sense for any.